bloom-housing/bloom
# 1320/additional eligibility, listings management

# Pull Request Template

## Issue

Addresses #1320 

## Description

Adds the additional eligibility fields to the listings management prototype.

## Type of change


## How Can This Be Tested/Reviewed?

Add a new listing in partners at listings/add and use the new fields, save as draft, restart the app, and check out the fields in public. See them at listings/[id-of-listing-you-just-made] for the detail view as well in partners.

## Checklist:
